mods/ra/sequences/aircraft.yaml

```diff
@@ -1,6 +1,7 @@
 mig:
 	idle:
 		Facings: 16
+		InterpolatedFacings: 1024
 	icon: migicon
 
 yak:
```

Comment:
Looks like you forgot the Yak?

And if interpolation doesn't impact sheets/RAM usage, I think we should also use 1024 for helicopters (in both RA and TD) as well as any other TD aircraft with 32 facings, because the jittering is present there too, just less pronounced than with RA's 16-facing aircraft.

Reply:
Yes that was an oversight, have now added it to Yak as well. The reason it wasn't added to the helicopters is because the original Red Alert only had this rotation effect for 16 facing aircraft, and helicopters have 32 facings. It can still be added of course, but may be better to get some consensus on this since it deviates from the original game a bit. I have made an exception for the Carry-All from D2K for this jittering reason, so in my view it makes sense to do it for the helicopters as well if it improves the look of the animation.